src/forms/my.applications.js

```diff
@@ -86,22 +87,20 @@ class MyApplications extends Component {
     archive() {
         let idsToArchive = this.state.cases.reduce(this.onlySelected, []);
         if (idsToArchive.length > 0) {
-            this.window.document.getElementById('are-you-sure-modal').style.display = 'block';
+            this.setState({ displayAreYourSureDialog:true });
```

Comment:
as a style thing, I think we should include a space between the colon and the next thing.  displayAreYourSureDialog: true.  Also you have a typo... Are You sure, not Are YouR Sure.